web/locales/zh-hk.json

```
@@ -254,7 +254,7 @@
             "zoneName": "黑山共和國"
           },
           "MK": {
-            "zoneName": "馬其頓共和國"
+            "zoneName": "北马其顿"
```

I'm not entirely sure, but it looks like the Traditional Chinese script name (as I guess would be used for zh-hk and zh-tw below) is 北馬其頓. Would you mind updating it in those two files?

```suggestion
            "zoneName": "北馬其頓"
```